common/index.ts

```diff
@@ -0,0 +1,40 @@
+import AdmZip from 'adm-zip';
+import fs from 'fs';
+import fetch from 'isomorphic-fetch';
+
+export enum ProjectSource {
+	FILE = 'file',
+	URI = 'uri',
+	CLOUD_ID = 'cloudId',
+	PROJECT_JSON = 'project.json',
+	MIT_PROJECT_URI = 'https://projects.scratch.mit.edu'
+}
+
+/*
+  The AssetFetcher is responsible for handing different
+  project sources and the asset files within or referred
+  to in a project.
+*/
+export const AssetFetcher = {
```

Comment:
I think this will need to be, if not this in PR sometime later, a configurable property for factories and not a singleton.  Of key importance I think is finding cloud id projects. There is no one url to rule them all. `'https://projects.scratch.mit.edu'` didn't exist to my knowledge for most of sb3's lifespan. It was added a month or two before release. So it could change in the future or sb-util users may want to configure a proxy.